examples/tabular/example_advanced_tabular.py

```diff
@@ -14,13 +14,15 @@
 savedir = 'ag_hpo_models/' # where to save trained models
 
 hyperparams = {'NN': {'num_epochs': 10, 'activation': 'relu', 'dropout_prob': ag.Real(0.0,0.5)},
-               'GBM': {'num_boost_round': 1000, 'learning_rate': ag.Real(0.01,0.1,log=True)} }
+               'GBM': {'num_boost_round': 1000, 'learning_rate': ag.Real(0.01,0.1,log=True)},
+               'XGB': {'n_estimators': 1000, 'learning_rate': ag.Real(0.01,0.1,log=True)} }
```

Comment:
Not sure why we would specifically add XGboost to this example. The example is just supposed to illustrate how users can exert more control over fit(), not to highlight what models are available.

Instead I think we should add a dedicated unit test that evaluates just the XGBoost model alone. @Innixma what do you think?

Reply:
Ideally we want a unit test for all of our models, so I think thats something that we can do after the PR to avoid delays in merging. Regarding example_advanced_tabular.py, I think it can be reverted to be unchanged, and a dedicated XGB test can be added in future (along with all the other models).